pkg/generic/ensure_release_label.go

```diff
@@ -26,7 +26,7 @@ func EnsureReleaseVersionLabel(ctx context.Context, ctrlClient client.Client, me
 		}
 
 		// Get release from Cluster.
-		release, err := getLabelValueFromCluster(ctx, ctrlClient, meta, label.ReleaseVersion)
+		release, err := getLabelValueFromAzureCluster(ctx, ctrlClient, meta, label.ReleaseVersion)
```

Comment:
Update comment above and log message below.

Reply:
done